cli/node/injector.go

```diff
@@ -29,6 +29,10 @@ func (inj *reflectInjector) Resolve(v interface{}) error {
 		return xerrors.New("expect a pointer")
 	}
 
+	if !rv.Elem().IsValid() {
+		return xerrors.Errorf("reflect value '%v' is invalid", rv)
```

Comment:
Can you complete the unit test for that statement? I think the `%v` might be too verbose.